api/src/test/java/marquez/service/LineageServiceTest.java

```diff
@@ -268,7 +268,7 @@ public void testLineageWithDeletedDataset() {
         .hasSize(0);
     runAssert
         .extracting(Run::getOutputVersions, InstanceOfAssertFactories.list(DatasetVersionId.class))
-        .hasSize(1);
+        .hasSize(0);
```

Comment:
I think this is a little misleading since we're not asking the database for this information anymore. Should we just remove this assertion or change the type definition?

Reply:
I agree and had initially removed this completely but then added it back again, because getInputVersions also had an assertion. I guess i should remove both of them now.